NuRadioMC/SignalProp/analyticraytracing.py

```diff
@@ -2083,6 +2555,23 @@ def apply_propagation_effects(self, efield, i_solution):
         if self._config['propagation']['focusing']:
             focusing = self.get_focusing(i_solution, limit=float(self._config['propagation']['focusing_limit']))
             spec[1:] *= focusing
+        
+        # apply the birefringence effect
+        if self._config['propagation']['birefringence']:
+        
+            try:
```

Comment:
bad style. If RadioPropa should be used or not should be a config flag.

Reply:
Done.